app/src/main/res/layout/activity_launch_screen.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".LaunchScreenActivity">
+
+    <TextView
+        android:id="@+id/txt_sign"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="37dp"
+        android:text="@string/launch_screen_txt_company_name"
+        android:textColor="@color/s_60"
+        android:textSize="14sp"
+        app:layout_constraintEnd_toEndOf="@+id/button"
+        app:layout_constraintTop_toTopOf="parent"
+        android:fontFamily="@font/poppins_regular"
+        />
+
+    <TextView
+        android:id="@+id/txt_welcome"
+        android:layout_width="265dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="81dp"
+        android:text="@string/launch_screen_txt_welcome"
+        android:textAlignment="center"
+        android:textColor="@color/s_60"
+        android:textSize="20sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:lineHeight="20sp"
+        android:fontFamily="@font/comfortaa_regular"
+        android:textStyle="bold"/>
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="24dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/txt_welcome"
+        app:srcCompat="@drawable/launcher_image"
+        android:contentDescription="@string/launch_screen_image_launch" />
+
+    <TextView
+        android:id="@+id/txt_message"
+        android:layout_width="196dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="18dp"
+        android:text="@string/launch_screen_txt_subtext"
+        android:textAlignment="center"
+        android:textColor="@color/s_60"
+        android:textSize="16sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/imageView"
+        app:lineHeight="20sp"
+        android:textStyle="bold"
+        android:fontFamily="@font/comfortaa_regular"/>
+
+    <!-- TODO: add style for button-->
+    <Button
+        android:id="@+id/button"
+        android:layout_width="0dp"
+        android:layout_height="48dp"
```

Comment:
Height of this button seems to be wrong when app is launched + it is purple instead of green